source/manual/components.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: '#publishing-frontend'
+title: Components
+section: Frontend
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-09-01
+review_in: 6 months
+---
+
+Components are packages of template, style, behaviour and documentation that live in your application or in static.
+
+## Component guides
+
+Components in applications are documented in component guides using the [govuk_publishing_components gem](https://github.com/alphagov/govuk_publishing_components). This mounts a component guide at the path `/component-guide` in development, test and on Heroku review apps.
+
+Find components in these guides:
+
+* [static component guide](https://govuk-static.herokuapp.com/component-guide/)
```

Comment:
This link doesn't work yet - I assume that's okay?

Reply:
It'll work now.